### ISSUE TITLE
Improve error message when method calls missed some commas in Julia 1.10

### ISSUE THREAD

**Issue by Tortar** [JuliaLang/julia#52285]
In Julia 1.10-rc1:

```julia
julia> f(x, y, z) = x+y+z
f (generic function with 1 method)

julia> f(3,4 5)
ERROR: ParseError:
# Error @ REPL[2]:1:7
f(3,4 5)
#     ╙ ── Expected `)`
Stacktrace:
 [1] top-level scope
   @ none:1
```

Contrast this with Python 3.10 which instead:

```python
>>> def f(x,y,z): return x+y+z
... 
>>> f(3,4 5)
  File "<stdin>", line 1
    f(3,4 5)
        ^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
```

The error message of Python seems better, can we change the Julia message to something similar? Probably multiple dispatch makes this trickier though.

Indeed it was better in Julia 1.9

```julia
julia> f(3,4 5)
ERROR: syntax: missing comma or ) in argument list
Stacktrace:
 [1] top-level scope
   @ none:1
```

**Comment by Tortar** [comment]
Actually this was fixed in JuliaSyntax 0.4.8 and was a duplicate (https://github.com/JuliaLang/JuliaSyntax.jl/pull/397) so I think this can be closed.

Base version is still at 0.4.7 though so the issue remains

**Comment by Tortar** [comment]
Actually there is https://github.com/JuliaLang/julia/pull/53119 trying to bump the version
